filebeat/module/elasticsearch/server/config/log.yml

```diff
@@ -8,3 +8,8 @@ multiline:
   pattern: '^\[[0-9]{4}-[0-9]{2}-[0-9]{2}'
   negate: true
   match: after
+
+fields:
```

Comment:
This is meant as an example / suggestion on how we should add fields in the future. 

Instead of using `fields` I would prefer if we would have a processor for it. @urso WDYT?

Reply:
So we would remove fields completely and have a `addCustomFieldProcessor` or `addFieldProcessor`?

If this is the case I would be +1 to do it.

Reply:
Exactly. We could add the processor in the 6.x cycle and deprecated fields. Then it's replaced in 7.0. With a processor we could also define things like if fields should be overwritten etc.

Reply:
+1 for flexibility, can you create an issue for it?

Reply:
Done: https://github.com/elastic/beats/issues/7350